mmcv/transforms/processing.py

```diff
@@ -498,6 +505,43 @@ def _crop_seg_map(self, results: dict, bboxes: np.ndarray) -> None:
             img = mmcv.imcrop(results['gt_semantic_seg'], bboxes=bboxes)
             results['gt_semantic_seg'] = img
 
+    def _crop_bboxes(self, results: dict, bboxes: np.ndarray) -> None:
+        """Update bounding boxes according to CenterCrop.
+
+        Args:
+            results (dict): Result dict contains the data to transform.
+            bboxes (np.ndarray): Shape (4, ), location of cropped bboxes.
+        """
+        if 'gt_bboxes' in results:
+            offset_w = bboxes[0]
+            offset_h = bboxes[1]
+            bbox_offset = np.array([offset_w, offset_h, offset_w, offset_h])
+            gt_bboxes = results['gt_bboxes'] - bbox_offset
```

Comment:
might add some comments for `gt_bbox` shape and contain
like `# gt_bbox with shape [B, 4], gt_bbox[n] is [something]`?